.devcontainer/devcontainer.json

```diff
@@ -15,8 +15,9 @@
   "forwardPorts": [3000, 4000],
 
   // Use 'postCreateCommand' to run commands after the container is created.
-  "postCreateCommand": ".devcontainer/post-create.sh",
-  "waitFor": "postCreateCommand",
+  "postCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder}",
+  "postStartCommand": ".devcontainer/post-create.sh",
+  "waitFor": "postStartCommand",
```

Comment:
Can we remove the `waitFor` command here? I understand that this PR did not add this property, however, looks like creating a codespace with these PR changes are having storage issues.

Removing this property helps create and connect to a codespace successfully. :)

Reply:
> looks like creating a codespace with these PR changes are having storage issues.

That might actually be due to the changes to libretranslate not the `waitFor`. Previously it didn’t have access to the external network, so it would run in a crash loop because it couldn’t download the language models. If codespaces has space restrictions, maybe libretranslate should be commented out? Is there a way to mark that as an optional component of the devcontainer?